### PR TITLE
Update the syndesis-prometheus in redhat-rhmi-fuse to 10Gi

### PR DESCRIPTION
**Pull request by Rajagopalan-Ranganathan**
Jira link: https://issues.redhat.com/browse/INTLY-7188
# Description
Reconcile the syndesis-prometheus pvc to 10Gi from 1Gi. Should update
for already existing installations (upgrade) and new installations as well.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist
- [ ] This change requires a documentation update 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Verified independently on a cluster by reviewer

### CODE REVIEW

**Comment by HubertStefanski** [comment]
> /test e2e

e2e tests are broken due problems with quay, They won't work until it's back up :crying_cat_face:

**Comment by HubertStefanski** [comment]
Verified upgrade on my cluster, Good Job :+1: @Rajagopalan-Ranganathan 
image
image

File: pkg/products/fuse/reconciler.go [hunk 1]
```diff
@@ -322,6 +326,30 @@ func (r *Reconciler) reconcileCustomResource(ctx context.Context, rhmi *integrea
...
+			pvccr.Spec.AccessModes = []v1.PersistentVolumeAccessMode{v1.ReadWriteOnce}
+			pvccr.Spec.Resources.Requests = make(v1.ResourceList)
+			pvccr.Spec.Resources.Requests[v1.ResourceStorage] = resource.MustParse(syndesisPrometheusPVC)
+		} else {
+			pvccr.Spec.Resources.Requests[v1.ResourceStorage] = resource.MustParse(syndesisPrometheusPVC)
+		}
+		return nil
+	})
+	if err != nil {
+		return integreatlyv1alpha1.PhaseFailed, fmt.Errorf("failed to create or update syndesis-promtheus PVC custom resource: %v", err)
```
[the remark this answers is not in the document]
Comment (Rajagopalan-Ranganathan) [reply pkg/products/fuse/reconciler.go:347]: Sure. I will change it back to %w.

**Comment by david-martin** [comment]
/approve

@HubertStefanski if you could give an 👁️ again and lgtm?

**Comment by HubertStefanski** [comment]
Wait for #780 to get merged before retesting

**Pull request merged**